patronload/staff.py

```diff
@@ -61,7 +66,11 @@ def phone_format(n):
 # Start Oracle data import #
 
 # Connect to "WAREHOUSE.WORLD" as defined in the tns.ora
-connection = cx_Oracle.connect(dw['user'], dw['password'], "WAREHOUSE.WORLD")
+connection = cx_Oracle.connect(
+    data_warehouse_user,
+    data_warehouse_password,
+    "WAREHOUSE.WORLD"
```

Comment:
I suggest that we go ahead and replace the "warehouse.world" here with an SSM variable also, since we currently set that as a file on the filesystem in alma during setup.

Reply:
Agreed. Let's set host, port, and sid params and then we can [`cx_oracle.makedsn(host, port, sid)`](https://cx-oracle.readthedocs.io/en/latest/api_manual/module.html#cx_Oracle.makedsn) to get the dsn string.